Makefile

```diff
@@ -36,8 +36,7 @@ freeze-requirements:
 	echo '# This file is autogenerated. Do not edit it manually.' > requirements.txt
 	cat requirements-app.txt >> requirements.txt
 	echo '' >> requirements.txt
-	$$(pwd)/venv-freeze/bin/pip freeze -r requirements-app.txt | sed -n '/The following requirements were added by pip freeze/,$$p' >> requirements.txt
-	rm -rf venv-freeze
```

Comment:
The removal of `rm -rf venv-freeze` is intentional?

Reply:
As it also does this as the first step of `freeze-requirements` I guess there's no harm in leaving it hanging around at the end?

Reply:
Ah that should be put back please sorry.